src/spyglass/lfp/lfp_electrode.py

```diff
@@ -94,29 +94,77 @@ def create_lfp_electrode_group(
             "nwb_file_name": nwb_file_name,
             "lfp_electrode_group_name": group_name,
         }
-        electrode_restrictions = [
-            {"electrode_id": eid} for eid in electrode_list
-        ]
-        electrode_keys_to_insert = (
-            electrode_table & electrode_restrictions
-        ).fetch("KEY")
+
+        restriction_str = (
+            f"electrode_id = {electrode_list[0]}"
+            if len(electrode_list) == 1
+            else f"electrode_id in {tuple(electrode_list)}"
```

Comment:
Not certain, but I think `f"field in {tuple(some_int)}"` because it gets converted to `"field in (1)"`. Not worth changing